### PR TITLE
[MM-23670] Improve formatting of Channel header change message

### PR DESCRIPTION
**Pull request by lestgabo**
#### Summary
Make update header message easier to read.
- I took some liberties on where to put break lines in Japanese, Korean, Turkish, Chinese (simplified), and Chinese (Taiwan) 

#### Ticket Link
github issue was created at server repo -> Fixes https://github.com/mattermost/mattermost-server/issues/14256
jira ticket -> https://mattermost.atlassian.net/browse/MM-23670

#### Related Pull Requests
n/a

#### Screenshots
Screen Shot 2020-10-16 at 6 26 09 PM

### CODE REVIEW

**Comment by lestgabo** [comment]
I didnt run `npm run check` ill fix asap, I see where the lint failed.

File: i18n/en.json [hunk 1]
```diff
@@ -2073,7 +2073,7 @@
   "api.channel.post_convert_channel_to_private.updated_from": "{username} converted the channel from public to private",
   "api.channel.post_update_channel_displayname_message_and_forget.updated_from": "{username} updated the channel display name from: {old} to: {new}",
   "api.channel.post_update_channel_header_message_and_forget.removed": "{username} removed the channel header (was: {old})",
-  "api.channel.post_update_channel_header_message_and_forget.updated_from": "{username} updated the channel header from: {old} to: {new}",
+  "api.channel.post_update_channel_header_message_and_forget.updated_from": "{username} updated the channel header <br></br><strong>from:</strong> {old} <br></br><strong>to:</strong> {new}",
```
Comment (esethna) [inline i18n/en.json:2076]: @lestgabo can we please capitalize "To: and "From:"?

**Review by esethna — APPROVED** [review]

File: i18n/en.json (hunk 1 above)
Comment (lestgabo) [reply i18n/en.json:2076]: I wouldn't know how to capitalize correctly for other languages so I will only do so for the English version :)

Comment (lestgabo) [reply i18n/en.json]: @esethna done

**Review by esethna — APPROVED** [review]
Thanks!

**Review by calebroseland — APPROVED** [review]
LGTM! 🎉

Thanks for the proactiveness with i18n, though the non-`en` strings will eventually get overwritten by https://translate.mattermost.com/. If you wouldn't mind, it would probably be better to suggest them over there.

https://handbook.mattermost.com/contributors/contributors/localization

**Review by asaadmahmood — APPROVED** [review]

**Review by ogi-m — APPROVED** [review]
LGTM 👍

**Pull request merged**